examples/todo/src/index.ts

```diff
@@ -8,6 +8,13 @@ import {ApplicationConfig} from '@loopback/core';
 import {RestServer} from '@loopback/rest';
 
 export async function main(options?: ApplicationConfig) {
+  // IBM Cloud app port is set on process.env.PORT
+  if (!options) {
+    options = {rest: {port: process.env.PORT}};
```

Comment:
This isn't needed since the port default to `process.env.PORT` in the `else` statement below. So this can be just:
```ts
options.rest = options.rest ? options.rest : {};
options.rest.port = process.env.PORT || options.rest.port;
```

Reply:
Where is `options ` defined?